_episodes/12-cluster.md

```diff
@@ -139,6 +139,12 @@ node*, *login node*, *landing pad*, or *submit node*. A login node serves as an
 cluster. As a gateway, it is well suited for uploading and downloading files, setting up software,
 and running quick tests. It should never be used for doing actual work.
 
+> Dedicated Transfer Nodes
+>
+> If you want to transfer larger amounts of data to or from the cluster, some systems offer dedicated nodes for data transfers only. The motivation for this lies in the fact, that larger data transfers should not obstruct operation of the login node for anybody else. 
```

Comment:
These lines are causing warnings because they are too long, it's good for us to stick to the 100 character length rule because it will help with translation tracking later

Reply:
9a84ced